install_config/configuring_pvc_protection.adoc

```diff
@@ -22,13 +22,14 @@ PVC protection is an alpha feature and may change in a future release of {produc
 ====
 
 [[local-volume-enabling-pvc-protection]]
-=== Enable PVC Protection
+== Enabling PVC protection
 
 To enable the `PVCProtection` feature gate on all masters and nodes:
 
 . Edit or create the master configuration file on all masters (*_/etc/origin/master/master-config.yaml_* by default):
+.. Add `PVCProtection` under the `admissionConfig` section.
```

Comment:
From OCP 3.10, PVCProtection is a beta feature, do not need enable feature gate manually.

Reply:
Yes, this file will be removed in the 3.10 docs. So, I will not make any additional changes to it.